core/pom.xml

```diff
@@ -468,6 +468,11 @@ THE SOFTWARE.
       <artifactId>commons-logging</artifactId>
     </dependency>
     <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.17</version>
```

Comment:
Prefered way is define version in properties section, but @jglick comments/review will be welcome.